drivers/adc/ad7193/ad7193.c

```diff
@@ -96,6 +93,21 @@ int8_t ad7193_init(struct ad7193_dev **device,
 		status = -1;
 	}
 
+	/* Reset */
+	AD7193_CS_LOW;
```

Comment:
since we update de driver, I'd remove these definitions: https://github.com/analogdevicesinc/no-OS/blob/cbd8405892eeadc4383a6d8ddc3644fb9a6f4743/drivers/adc/ad7193/ad7193.h#L58-L65 in a separate commit and use gpio getters/setters inline. But that's your call 😄

Reply:
Will, update this as well.